src/resolve/resolved-dns-transaction.h

```diff
@@ -20,6 +20,7 @@ enum DnsTransactionState {
         DNS_TRANSACTION_PENDING,
         DNS_TRANSACTION_VALIDATING,
         DNS_TRANSACTION_RCODE_FAILURE,
+        DNS_TRANSACTION_EDE_RCODE_FAILURE,
```

Comment:
this state is only about dnssec right? so it should carry that in the name somehow

Reply:
I only consider the dnssec failure modes atm. I guess if there are others they can always get their own state value.

Reply:
Changed this name to reflect that it is only used in DNSSEC.